automatic/jenkins/update.ps1

```diff
@@ -0,0 +1,42 @@
+#requires -Modules au, Wormies-AU-Helpers
+param(
+    # The URL to search for information on the latest release
+    $ReleasePage = "https://www.jenkins.io/download/",
+
+    # The regex used to search for the latest version on the ReleasePage
+    $ReleaseRegex = "Download Jenkins`n(?<Version>[\d\.]+)`nLTS",
+
+    # The base of the download URL
+    $DownloadURL = "https://mirrors.jenkins-ci.org/windows-stable/"
+)
```

Comment:
Please follow similar style used in other update scripts.

```suggestion
Import-Module au

$ReleasePage = "https://www.jenkins.io/download/",
$ReleaseRegex = "Download Jenkins`n(?<Version>[\d\.]+)`nLTS",
$DownloadURL = "https://mirrors.jenkins-ci.org/windows-stable/"
```

We automatically import `Wormie-AU-Helpers`, so don't import this one manually here (at least not when checking in the file).